project/webapp/templates/thread.html

```diff
@@ -6,6 +6,24 @@
     {% include "base/less_headers/thread_less.html" %}
 </head>
 
+<script>
+$(document).ready(function() {
+  $('#csv-export').click(function() {
+    $.ajax({
+      type: 'POST',
+      url: 'http://127.0.0.1:8000/civi2csv',
+      data: JSON.stringify({'thread': /[^/]*$/.exec(window.location.href)[0]})
```

Comment:
What does this line do after 'thread'? Particularly, why do we need to use `exec(...`?